frontend/test/specs/notification/notificationControllerSpec.js

```diff
@@ -62,6 +62,8 @@
         };
         notCtrl = createCtrl();
         notCtrl.notifications = [];
+
+        expect(notCtrl.toolbarMobileMenuItems.length).toEqual(1);
```

Comment:
Why this expect are alone in the beforeEach?

Reply:
This field is set when the controller is instantiated, so, before each test I expect it has this length..